Gemfile

```diff
@@ -25,7 +25,9 @@ gem 'resque-timeout', '~> 1.0.0'
 gem 'resque-lock-timeout', '~> 0.4.5'
 gem 'resque-scheduler', '~> 4.3.1'
 # Paperclip is deprecated: https://cm-jira.usa.gov/browse/SRCH-702
-gem 'paperclip', '~> 5.2.0'
+# Using a third-party fork as an interim measure.
+gem 'kt-paperclip', :git => 'https://github.com/kreeti/kt-paperclip.git'
+gem 'aws-sdk', '~> 3.0.2'
```

Comment:
This is adding a boatload of gems that we don't need as dependencies. Per the docs, "Version 3 of the AWS SDK for Ruby modularizes the monolithic SDK into service-specific gems, for example, aws-sdk-s3 and aws-sdk-dynamodb." kt-paperclip specifies that we only need the `aws-sdk-s3` gem: https://github.com/kreeti/kt-paperclip#understanding-storage

Reply:
Changed; thanks for the catch.